test/source-test.js

```diff
@@ -21,3 +22,31 @@ test('resolve() converts markdown to html', t => {
     t.truthy(result[0].endsWith('.html'))
   })
 })
+
+test('resolve() includes custom css file', async t => {
+  t.plan(1)
+
+  var result = await source.resolve(['./README.md'], { css: 'custom.css' })
+  var data = await fs.readFile(result[0], 'utf-8')
+
+  t.regex(data, /.*custom\.css.*/g)
+})
+
+test('resolve() includes multiple custom css files', async t => {
+  t.plan(2)
+
+  var result = await source.resolve(['./README.md'], { css: ['first.css', 'second.css'] })
+  var data = await fs.readFile(result[0], 'utf-8')
```

Comment:
You are correct, files are created. But:

- technically only _during_ the tests and not _by_ the tests
- the framework doesn't remove them either 
- the existing tests didn't remove them either

I think it's not a very big deal because they are created in the temporary location. Maybe the framework should use a library like [tmp](https://github.com/raszi/node-tmp) to clean-up after itself. But that would suit for another issue.

Or like they say at [tempy](https://www.npmjs.com/package/tempy#why-doesnt-it-have-a-cleanup-method):
> The operating system will clean up when needed. No point in us wasting resources and adding complexity.